src/category_theory/additive/basic.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2021 Luke Kershaw. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Luke Kershaw
+-/
+import category_theory.limits.constructions.pullbacks
+import category_theory.limits.shapes.biproducts
+import category_theory.limits.shapes.images
+import category_theory.abelian.non_preadditive
+import category_theory.shift
+
+/-!
+# Additive Categories
+
+This file contains the definition of additive categories.
+
```

Comment:
A TODO here would be to show that finite biproducts implies enriched over commutative monoids and what is missing additionally to have additivity is that identities have additive inverses, see https://ncatlab.org/nlab/show/biproduct#SemiadditivityAsStructureProperty